cmd/clairctl/.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: go
+go:
+- 1.5
```

Comment:
We should do tip and 1.6 as well if possible. This is non-blocker though, we can do it later.